pyquil/gates.py

```diff
@@ -382,7 +381,14 @@ def CPHASE10(angle: ParameterDesignator, control: QubitDesignator, target: Qubit
     return Gate(name="CPHASE10", params=[angle], qubits=qubits)
 
 
-def CPHASE(angle: ParameterDesignator, control: QubitDesignator, target: QubitDesignator) -> Gate:
+# NOTE: We don't use ParameterDesignator here because of the following Sphinx error
+#   Cannot resolve forward reference in type annotations of "pyquil.gates.CPHASE":
+#   name 'Expression' is not defined
```

Comment:
> This error can be resolved by importing `Expression`, but then flake8 complains about an unused import...

or something to that effect. Otherwise the comment doesn't fully explain the chicken-and-egg situation or why you can't solve it by just importing `Expression`